.github/workflows/ci.yml

```
@@ -80,4 +80,4 @@ jobs:
             fail-fast: false
             matrix:
                 os: [ubuntu-latest, windows-latest, macos-latest]
-                php-version: ['8.1']
+                php-version: ['8.2']
```

Should still be testing on 8.1.
```suggestion
                php-version: ['8.1', '8.2']
```

---

And 8.2 should be marked as experimental